docs/rfds/acp-agent-registry.mdx

```diff
@@ -0,0 +1,125 @@
+| `homepage`       | URL for docs/marketing.                                                                                                                                                                                                                                                                                                                                            |
+| `repository`     | Source repository URL.                                                                                                                                                                                                                                                                                                                                             |
+| `authors`        | Array of author/organization names (mirrors `authors` in the TOML example).                                                                                                                                                                                                                                                                                        |
+| `license`        | SPDX identifier or `"proprietary"`.                                                                                                                                                                                                                                                                                                                                |
+| `capabilities`   | Array of ACP method names implemented (e.g. `["terminal/new","files/read"]`).                                                                                                                                                                                                                                                                                      |
+| `auth`           | Array of auth options for authentication. This is the trickiest part of the schema.                                                                                                                                                                                                                                                                                |
+| `distribution`   | _Optional._ Object mapping target platforms to download/execution info. Each target key follows `<os>-<arch>` format (e.g., `darwin-aarch64`, `linux-x86_64`, `windows-x86_64`). Each target specifies `archive` (download URL), `cmd` (executable path), optional `args` (array of command-line arguments), and optional `env` (object of environment variables). |
```

Comment:
food for thought, but I think the distribution manifest could go in many ways.. we may not be able to assume a latest tag for things like this. I ran into this with envoy and here's an example registry where ended up needing to add a date field (though many agents update multiple times a day, envoy never) https://archive.tetratelabs.io/envoy/envoy-versions.json

main concern about the download manifest part is if this digs a hole quite deep. it could be that we make a conventional manifest that is independent from the registry, then the registry assumes that packaing? just ideas.